codesearch/index/write.go

```diff
@@ -563,15 +563,16 @@ func (b *bufWriter) offset() uint32 {
 	return uint32(off)
 }
 
-func (b *bufWriter) flush() {
+func (b *bufWriter) flush() error {
```

Comment:
Should we care that the other callers of flush() besides Flush() are not handling the returned error?

<img width="648" alt="flush_callers" src="https://user-images.githubusercontent.com/214201/102922161-6a7c6680-4442-11eb-86ec-171fb7d75a3b.png">